homeassistant/components/tado/switch.py

```diff
@@ -0,0 +1,94 @@
+"""Module for Tado child lock switch entity."""
+
+import logging
+from typing import Any
+
+from homeassistant.components.switch import SwitchEntity
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import TadoConfigEntry
+from .entity import TadoDataUpdateCoordinator, TadoZoneEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class TadoChildLockSwitchEntity(TadoZoneEntity, SwitchEntity):
+    """Representation of a Tado child lock switch entity."""
+
+    def __init__(
+        self,
+        coordinator: TadoDataUpdateCoordinator,
+        zone_name: str,
+        zone_id: int,
+        device_info: dict[str, Any],
+    ) -> None:
+        """Initialize the Tado child lock switch entity."""
+        super().__init__(zone_name, coordinator.home_id, zone_id, coordinator)
+
+        self._device_info = device_info
+        self._device_id = self._device_info["shortSerialNo"]
+        self._attr_unique_id = f"{zone_id} {coordinator.home_id} child-lock"
```

Comment:
what do we use as unique identifier seperator in the other platforms? Also a space?

Reply:
From `climate.py`:

```python
self._attr_unique_id = f"{zone_type} {zone_id} {coordinator.home_id}"
```

Reply:
From `water_heater.py`

```python
self._attr_unique_id = f"{zone_id} {coordinator.home_id}"
```